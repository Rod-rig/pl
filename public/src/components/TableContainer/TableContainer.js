import React from 'react';
import table from 'Services/england-table';
import Row from '../Row/Row';

class TableContainer extends React.Component {
    constructor(props) {
        super(props);
        this.state = {
            table: [...table],
            sort: 'position'
        };
        this.sort = (e) => {
            this.setState({
                sort: e.target.textContent
            });
        };
    }

    render() {
        return (
            <div>
                <table>
                    <thead>
                        <tr onClick={this.sort}>
                            {
                                Object.keys(table[0]).map((title, index) => <td key={index}>{title}</td>)
                            }
                        </tr>
                    </thead>
                    <tbody>
                        {
                            table
                                .sort((a, b) => b[this.state.sort] > a[this.state.sort])
                                .map((row, index) => <Row
                                    key={index}
                                    row={row}/>)
                        }
                    </tbody>
                </table>
            </div>
        );
    }
}

export default TableContainer;
