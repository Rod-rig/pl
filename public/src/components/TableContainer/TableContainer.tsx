import axios, {AxiosResponse} from 'axios';
import {Table, TableBody} from 'material-ui';
import * as React from 'react';
import {IRow, Row} from '../Row/Row';
import TableHeadContainer from '../TableHeadContainer/TableHeadContainer';

const ghUrl: string = 'https://raw.githubusercontent.com/Rod-rig/epl-data/master';

interface IState {
  order: 'asc' | 'desc';
  sort: string;
  table: object[];
}

interface IProps {
  chars: string[];
  id?: string;
  match?: {
    params: {
      id: string;
    };
  };
  range?: number[];
}

class TableContainer extends React.Component<IProps, IState> {
  private readonly id: string;
  private readonly range: number[] | null;
  constructor(props: IProps) {
    super(props);
    this.id = this.props.match ? this.props.match.params.id : this.props.id;
    this.range = this.props.range ? this.props.range : null;
    this.state = {
      order: 'asc',
      sort: 'position',
      table: [],
    };
    this.sort = this.sort.bind(this);
  }

  public sort(e: any) {
    const key = e.target.textContent;
    if (this.state.sort === key) {
      this.setState({
        order: this.state.order === 'asc' ? 'desc' : 'asc',
      });
    } else {
      this.setState({
        order: 'asc',
        sort: e.target.textContent,
      });
    }
  }

  public rangeData(data: any[], from: number, to: number) {
    if (from > to) {
      [from, to] = [to, from];
    }
    const dataSize: number = data.length;
    if (from > dataSize) {
      return data;
    }
    from = from > 0 ? from - 1 : 0;
    to = to < dataSize ? to : dataSize;
    return data.slice(from, to);
  }

  public componentDidMount() {
    const tableUrl = `${ghUrl}/2017-2018/england/${this.id}/table.json`;
    axios.get(tableUrl)
      .then((res: AxiosResponse) => {
        this.setState({
          table: this.range ? [...this.rangeData(res.data, this.range[0], this.range[1])] : [...res.data],
        });
      })
      .catch(/* istanbul ignore next */(error) => {
        throw error;
      });
  }

  public renderBody(rows: object[], chars: string[]): JSX.Element[] {
    return rows.map((row: IRow, index: number): JSX.Element => (
      <Row
        key={index}
        row={row}
        chars={chars}
      />
    ));
  }

  public render() {
    const chars = this.props.chars;
    const state = this.state;
    const sortedTable = state.table.sort((a: any, b: any) => {
      if (this.state.order === 'asc') {
        return a[state.sort] - b[state.sort];
      } else {
        return b[state.sort] - a[state.sort];
      }
    });

    return (
      <div>
        <Table>
          <TableHeadContainer
            order={state.order}
            sort={state.sort}
            sortHandle={this.sort}
            chars={chars}
          />
          <TableBody>{this.renderBody(sortedTable, chars)}</TableBody>
        </Table>
      </div>
    );
  }
}

export default TableContainer;
