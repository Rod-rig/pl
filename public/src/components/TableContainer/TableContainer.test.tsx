import {shallow} from 'enzyme';
import * as React from 'react';
import TableContainer from './TableContainer';

describe('TableContainer', () => {
  jest.mock('axios');
  const tableContainer = shallow(
    <TableContainer
      chars={['test']}
      match={{params: {id: 'premier-league'}}}
    />);

  it('should have head', () => {
    const head = tableContainer.find('TableHeadContainer');
    expect(head).toHaveLength(1);
    expect(head.prop('order')).toEqual('asc');
  });

  it('should have correct initial state', () => {
    expect(tableContainer.state('sort')).toEqual('position');
    expect(tableContainer.state('order')).toEqual('asc');
  });
});

describe('sorting in TableContainer', () => {
  const tableContainer = shallow(<TableContainer chars={['test']} id='premier-league'/>);
  const tableInst = tableContainer.instance();
  const sortSpy = jest.fn(TableContainer.prototype.sort.bind(tableInst));

  it('should change order state if sort was not changed', () => {
    sortSpy({target: {textContent: tableContainer.state('sort')}});
    expect(sortSpy).toHaveBeenCalledTimes(1);
    expect(tableContainer.state('order')).toEqual('desc');
  });

  it('should set default order if sort state was repeated', () => {
    sortSpy({target: {textContent: tableContainer.state('sort')}});
    expect(sortSpy).toHaveBeenCalledTimes(2);
    expect(tableContainer.state('order')).toEqual('asc');
  });

  it('should change sort state if we get custom params', () => {
    sortSpy({target: {textContent: 'foo'}});
    expect(sortSpy).toHaveBeenCalledTimes(3);
    expect(tableContainer.state('order')).toEqual('asc');
    expect(tableContainer.state('sort')).toEqual('foo');
  });
});
