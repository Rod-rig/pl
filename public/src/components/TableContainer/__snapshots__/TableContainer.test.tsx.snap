// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableContainer should render correctly 1`] = `
<div>
  <table
    className="MuiTable-root-1"
  >
    <thead>
      <tr
        className="MuiTableRow-root-2 MuiTableRow-typeHead-3"
      >
        <th
          aria-sort="descending"
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeHead-9"
          scope="col"
        >
          <div
            className="MuiTooltip-root-15"
          >
            <span
              aria-describedby={undefined}
              className="MuiButtonBase-root-29 MuiTableSortLabel-root-24 MuiTableSortLabel-active-25"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex="0"
            >
              position
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root-31 MuiTableSortLabel-icon-26 MuiTableSortLabel-asc-28"
                color={undefined}
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
            <div
              className="MuiTooltip-popper-16 MuiTooltip-popperClose-17"
              data-placement={undefined}
              data-x-out-of-boundaries={undefined}
              style={
                Object {
                  "left": 0,
                  "opacity": 0,
                  "pointerEvents": "none",
                  "position": "absolute",
                  "top": 0,
                }
              }
            >
              <div
                aria-hidden={true}
                className="MuiTooltip-tooltip-18 MuiTooltip-tooltipBottom-22"
                id={undefined}
                role="tooltip"
              >
                Sort
              </div>
            </div>
          </div>
        </th>
        <th
          aria-sort="descending"
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeHead-9"
          scope="col"
        >
          <div
            className="MuiTooltip-root-15"
          >
            <span
              aria-describedby={undefined}
              className="MuiButtonBase-root-29 MuiTableSortLabel-root-24"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex="0"
            >
              teamName
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root-31 MuiTableSortLabel-icon-26 MuiTableSortLabel-asc-28"
                color={undefined}
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
            <div
              className="MuiTooltip-popper-16 MuiTooltip-popperClose-17"
              data-placement={undefined}
              data-x-out-of-boundaries={undefined}
              style={
                Object {
                  "left": 0,
                  "opacity": 0,
                  "pointerEvents": "none",
                  "position": "absolute",
                  "top": 0,
                }
              }
            >
              <div
                aria-hidden={true}
                className="MuiTooltip-tooltip-18 MuiTooltip-tooltipBottom-22"
                id={undefined}
                role="tooltip"
              >
                Sort
              </div>
            </div>
          </div>
        </th>
        <th
          aria-sort="descending"
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeHead-9"
          scope="col"
        >
          <div
            className="MuiTooltip-root-15"
          >
            <span
              aria-describedby={undefined}
              className="MuiButtonBase-root-29 MuiTableSortLabel-root-24"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex="0"
            >
              matches
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root-31 MuiTableSortLabel-icon-26 MuiTableSortLabel-asc-28"
                color={undefined}
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
            <div
              className="MuiTooltip-popper-16 MuiTooltip-popperClose-17"
              data-placement={undefined}
              data-x-out-of-boundaries={undefined}
              style={
                Object {
                  "left": 0,
                  "opacity": 0,
                  "pointerEvents": "none",
                  "position": "absolute",
                  "top": 0,
                }
              }
            >
              <div
                aria-hidden={true}
                className="MuiTooltip-tooltip-18 MuiTooltip-tooltipBottom-22"
                id={undefined}
                role="tooltip"
              >
                Sort
              </div>
            </div>
          </div>
        </th>
        <th
          aria-sort="descending"
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeHead-9"
          scope="col"
        >
          <div
            className="MuiTooltip-root-15"
          >
            <span
              aria-describedby={undefined}
              className="MuiButtonBase-root-29 MuiTableSortLabel-root-24"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex="0"
            >
              w
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root-31 MuiTableSortLabel-icon-26 MuiTableSortLabel-asc-28"
                color={undefined}
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
            <div
              className="MuiTooltip-popper-16 MuiTooltip-popperClose-17"
              data-placement={undefined}
              data-x-out-of-boundaries={undefined}
              style={
                Object {
                  "left": 0,
                  "opacity": 0,
                  "pointerEvents": "none",
                  "position": "absolute",
                  "top": 0,
                }
              }
            >
              <div
                aria-hidden={true}
                className="MuiTooltip-tooltip-18 MuiTooltip-tooltipBottom-22"
                id={undefined}
                role="tooltip"
              >
                Sort
              </div>
            </div>
          </div>
        </th>
        <th
          aria-sort="descending"
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeHead-9"
          scope="col"
        >
          <div
            className="MuiTooltip-root-15"
          >
            <span
              aria-describedby={undefined}
              className="MuiButtonBase-root-29 MuiTableSortLabel-root-24"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex="0"
            >
              d
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root-31 MuiTableSortLabel-icon-26 MuiTableSortLabel-asc-28"
                color={undefined}
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
            <div
              className="MuiTooltip-popper-16 MuiTooltip-popperClose-17"
              data-placement={undefined}
              data-x-out-of-boundaries={undefined}
              style={
                Object {
                  "left": 0,
                  "opacity": 0,
                  "pointerEvents": "none",
                  "position": "absolute",
                  "top": 0,
                }
              }
            >
              <div
                aria-hidden={true}
                className="MuiTooltip-tooltip-18 MuiTooltip-tooltipBottom-22"
                id={undefined}
                role="tooltip"
              >
                Sort
              </div>
            </div>
          </div>
        </th>
        <th
          aria-sort="descending"
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeHead-9"
          scope="col"
        >
          <div
            className="MuiTooltip-root-15"
          >
            <span
              aria-describedby={undefined}
              className="MuiButtonBase-root-29 MuiTableSortLabel-root-24"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex="0"
            >
              l
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root-31 MuiTableSortLabel-icon-26 MuiTableSortLabel-asc-28"
                color={undefined}
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
            <div
              className="MuiTooltip-popper-16 MuiTooltip-popperClose-17"
              data-placement={undefined}
              data-x-out-of-boundaries={undefined}
              style={
                Object {
                  "left": 0,
                  "opacity": 0,
                  "pointerEvents": "none",
                  "position": "absolute",
                  "top": 0,
                }
              }
            >
              <div
                aria-hidden={true}
                className="MuiTooltip-tooltip-18 MuiTooltip-tooltipBottom-22"
                id={undefined}
                role="tooltip"
              >
                Sort
              </div>
            </div>
          </div>
        </th>
        <th
          aria-sort="descending"
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeHead-9"
          scope="col"
        >
          <div
            className="MuiTooltip-root-15"
          >
            <span
              aria-describedby={undefined}
              className="MuiButtonBase-root-29 MuiTableSortLabel-root-24"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex="0"
            >
              goals
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root-31 MuiTableSortLabel-icon-26 MuiTableSortLabel-asc-28"
                color={undefined}
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
            <div
              className="MuiTooltip-popper-16 MuiTooltip-popperClose-17"
              data-placement={undefined}
              data-x-out-of-boundaries={undefined}
              style={
                Object {
                  "left": 0,
                  "opacity": 0,
                  "pointerEvents": "none",
                  "position": "absolute",
                  "top": 0,
                }
              }
            >
              <div
                aria-hidden={true}
                className="MuiTooltip-tooltip-18 MuiTooltip-tooltipBottom-22"
                id={undefined}
                role="tooltip"
              >
                Sort
              </div>
            </div>
          </div>
        </th>
        <th
          aria-sort="descending"
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeHead-9"
          scope="col"
        >
          <div
            className="MuiTooltip-root-15"
          >
            <span
              aria-describedby={undefined}
              className="MuiButtonBase-root-29 MuiTableSortLabel-root-24"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex="0"
            >
              points
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root-31 MuiTableSortLabel-icon-26 MuiTableSortLabel-asc-28"
                color={undefined}
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
            <div
              className="MuiTooltip-popper-16 MuiTooltip-popperClose-17"
              data-placement={undefined}
              data-x-out-of-boundaries={undefined}
              style={
                Object {
                  "left": 0,
                  "opacity": 0,
                  "pointerEvents": "none",
                  "position": "absolute",
                  "top": 0,
                }
              }
            >
              <div
                aria-hidden={true}
                className="MuiTooltip-tooltip-18 MuiTooltip-tooltipBottom-22"
                id={undefined}
                role="tooltip"
              >
                Sort
              </div>
            </div>
          </div>
        </th>
      </tr>
    </thead>
    <tbody>
      <tr
        className="MuiTableRow-root-2 MuiTableRow-hover-6"
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          1.
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          Манчестер Сити
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          24
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          21
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          2
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          1
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          70:18
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          65
        </td>
      </tr>
      <tr
        className="MuiTableRow-root-2 MuiTableRow-hover-6"
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          2.
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          Манчестер Юнайтед
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          24
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          16
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          5
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          3
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          49:16
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          53
        </td>
      </tr>
      <tr
        className="MuiTableRow-root-2 MuiTableRow-hover-6"
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          3.
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          Челси
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          24
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          15
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          5
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          4
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          45:16
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root-7 MuiTableCell-paddingDefault-12 MuiTableCell-typeBody-10"
          scope={undefined}
        >
          50
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
